### ISSUE TITLE
JSON Fields are not properly parsed

### ISSUE THREAD

**Issue by dlitvakb** [contentful/contentful.rb#96]
Current workaround:

```ruby
entry.raw['fields']['yourJsonField']
```

Will output the expected result

**Comment by sbonami** [comment]
After spending a few minutes, I narrowed it down to about [this](https://github.com/contentful/contentful.rb/blob/fc5cc71ebd985d536857f1d961ac5e3f77062d7c/lib/contentful/resource.rb#L181) line for us.

**Comment by dlitvakb** [comment]
Hey @sbonami,

Thanks for the input, it really helps in tracking down the issue

Cheers

**Comment by dlitvakb** [comment]
This has been solved with #99, though JSON parsing is still not working as expected:

This example entry:

```json
{
  "fields": {
    "name": {"en-US": "Test"},
    "objectTest": {
    "en-US": {
      "null": null,
      "text": "some text",
      "array": [1, 2, 3],
      "number": 123,
      "object": {
        "null": null,
        "text": "bar",
        "array": [1, 2, 3],
        "number": 123,
        "object": {"foo": "bar"},
        "boolean": false},
      "boolean": true
    }
  }
}
```

Is being resolved as:

```ruby
[17] pry(main)> entry.fields
=> {:name=>"Test",
 :objectTest=>
  {:null=>"",
   :text=>"some text",
   :array=>"[1, 2, 3]",
   :number=>123,
   :object=>
    {"null"=>nil,
     "text"=>"bar",
     "array"=>[1, 2, 3],
     "number"=>123,
     "object"=>{"foo"=>"bar"},
     "boolean"=>false},
   :boolean=>"true"}}
```

Which has an incorrect value for the non-nested `array` field, and also nested object keys are stringified instead of symbols.

**Comment by dlitvakb** [comment]
Renaming the issue to JSON Fields are not properly parsed, and continuing to find a solution.
